modules/tensor_mechanics/doc/content/modules/tensor_mechanics/beam_vandv.md

```diff
@@ -0,0 +1,254 @@
+# Beams
+
+## Small strain Timoshenko beam bending
+
+A 1D cantilever beam of 4 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.554 ${m}^2$, moment of inertia about Y and Z axis of 0.0142 ${m}^4$, Young's modulus of elasticity of 2.6 N/${m}^2$, poisson's ratio of 0.3, shear modulus of 1 N/${m}^2$ and shear coefficient of 0.85. A point load of magnitude 1$\times$10$^-$$^4$ N is applied at the free end of the cantilever beam in Y-direction.
+
+!listing timoshenko_small_strain_y.i
+
+
+### Results
+
+For this beam, the dimensionless parameter alpha is given by:
+\begin{equation}
+alpha = \frac {kAGL^2}{EI} = 204.3734
+\end{equation}
+The value of alpha is not high  enough for the beam to behave like a thin beam where shear effects are not significant. Hence, the shear effects are considered and the small deformation analytical deflection of a cantilever beam is given by:
+\begin{equation}
+\Delta = \frac {PL^3}{3EI} \; (1 + \frac {3}{alpha}) = 5.868 \times 10^{-2} m
+\end{equation}
+
+The deflection obtained from MOOSE using 10 elements is 5.852$\times$10$^-$$^2$ m shown in [fig:timoshenko_small_strain2].
+
+!media media/tensor_mechanics/vandv/timoshenko_small_strain2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:timoshenko_small_strain2
+       caption=Displacement of the Timoshenko beam in bending.
+
+## Small strain Euler beam bending
+
+A 1D cantilever beam of 4 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.554 ${m}^2$, moment of inertia about Y and Z axis of 0.0142 ${m}^4$, Young's modulus of elasticity of 2.6 N/${m}^2$, poisson's ratio of -0.99, shear modulus of 1$\times$10$^4$ N/${m}^2$ and shear coefficient of 0.85. A point load of magnitude 1$\times$10$^-$$^4$ N is applied at the free end of the cantilever beam in Y-direction.
+
+!listing euler_small_strain_y.i
+
+### Results
+
+For this beam, the dimensionless parameter alpha is given by:
+\begin{equation}
+alpha = \frac {kAGL^2}{EI} = 2.04 \times 10^6
+\end{equation}
+Since the value of alpha is quite high, the beam behaves like a thin beam where shear effects are not significant. Hence, the small deformation analytical deflection of a cantilever beam is given by:
+\begin{equation}
+\Delta = \frac {PL^3}{3EI} \; (1 + \frac {3}{alpha}) = \frac {PL^3}{3EI} \; =5.78 \times 10^{-2} m
+\end{equation}
+
+The deflection obtained from MOOSE using 10 elements is 5.766$\times$10$^-$$^2$ m shown in [fig:euler_small_strain2]. The ratio beam FEM solution and analytical solution is 0.998.
```

Comment:
The second sentence could be rewritten as 'The ratio between the beam FEM solution and the analytical solution is 0.998' but I won't hold up merging this in for this sentence structure